clusterloader2/pkg/framework/framework.go

```diff
@@ -115,3 +150,15 @@ func (f *Framework) DeleteObject(gvk schema.GroupVersionKind, namespace string,
 func (f *Framework) GetObject(gvk schema.GroupVersionKind, namespace string, name string) (*unstructured.Unstructured, error) {
 	return client.GetObject(f.dynamicClient, gvk, namespace, name)
 }
+
+func isAutomanagedNamespace(name string) bool {
+	prefix := AutomanagedNamespaceName + "-"
```

Comment:
Maybe just `return regexp.MatchString(AutomanagedNamespaceName + "-[0-9]+", name)`?

Reply:
Done. I used `AutomanagedNamespaceName + "-[1-9][0-9]*"` as the patter, cause 1-based indexing.